packages/cli/src/prompts/choices.ts

```diff
@@ -0,0 +1,61 @@
+import { Separator } from 'inquirer';
+
+import difference from 'lodash.difference';
+import flatten from 'lodash.flatten';
+import ProjectFile from '../models/files/ProjectFile';
+import ContractManager from '../models/local/ContractManager';
+import Dependency from '../models/dependency/Dependency';
+
+export type Choice = string | { name: string; value: string } | { type: 'separator' };
+
+export type ContractsSource = 'built' | 'notAdded' | 'added' | 'all';
```

Comment:
My impression is that string literals are idiomatic. I couldn't find a source for this, but it's quite clear that the type system was designed around this idiom and it supports it really well. Both enums and symbols force you to import the value before using it, adding ceremony that isn't necessary and is prone to be worked around and cause errors. For example, if we use an enum it wouldn't be possible to use a string literal in its place as a function argument, because the string constants aren't subtypes of the enum, and it may lead a developer to write `'all' as ContractSource` which sneakily loses type safety since you can also do `'invalid' as ContractSource`.

Symbols are interesting but only add value when strong encapsulation is needed, since they allow to restrict access to a property to the module that created the symbol.